apps/game_backend/priv/repo/migrations/20240517093111_add_prestige_system_tables.exs

```diff
@@ -0,0 +1,16 @@
+defmodule GameBackend.Repo.Migrations.AddPrestigeSystemTables do
+  use Ecto.Migration
+
+  def change do
+    create table(:character_prestiges) do
+      add :user_id, references(:users, on_delete: :delete_all), null: false
+      add :character, :string, null: false
+      add :rank, :string, null: false
+      add :sub_rank, :integer, null: false
+      add :amount, :integer, null: false
+      timestamps()
+    end
```

Comment:
Let's be careful adding this. I think we should use the tables we already have.
In #637 I'm creating Units for Users.
Units have a 1-1 association with Character, and have attributes like rank.
I think we should use that if we'll share tables across games, but sharing some and creating our own seems like a mess.